src/Admin/SiteHealth.php

```diff
@@ -340,16 +349,28 @@ static function ( $available_service ) {
 	 *
 	 * @since 2.2.1
 	 *
+	 * @param int $threshold Threshold in milliseconds.
+	 *
 	 * @return int Threshold.
 	 */
-	public function get_good_response_time_threshold() {
-		/**
-		 * Filters the threshold below which a response time is considered good.
-		 *
-		 * @since 2.2.1
-		 * @param int $threshold Threshold in milliseconds.
-		 */
-		return (int) apply_filters( 'amp_page_cache_good_response_time_threshold', 600 );
+	public function get_good_response_time_threshold( $threshold = 600 ) {
+		if ( version_compare( get_bloginfo( 'version' ), '6.1', '>=' ) ) {
+			/** @deprecated 2.4.3 Use `site_status_good_response_time_threshold` instead. */
+			return (int) apply_filters_deprecated(
+				'amp_page_cache_good_response_time_threshold',
+				[ $threshold ],
+				'2.4.3',
+				'site_status_good_response_time_threshold'
+			);
+		} else {
+			/**
+			 * Filters the threshold below which a response time is considered good.
+			 *
+			 * @since 2.2.1
+			 * @param int $threshold Threshold in milliseconds.
+			 */
+			return (int) apply_filters( 'amp_page_cache_good_response_time_threshold', 600 );
```

Comment:
Not sure if this is the best approach to deprecate `amp_page_cache_good_response_time_threshold` but we need to deprecate toitfor WP >= 6.1.